general-configuration.md

```diff
@@ -66,13 +66,15 @@ The `resolveId()` method should return the **ID** of the currently logged `User`
 <?php
 namespace App;
 
+use Auth;
```

Comment:
Good spot, but I also prefer the FQCN instead of the alias.
So, it should instead be: `use Illuminate\Support\Facades\Auth;`
And it should go after the `Model` import.